.github/release-changelog-config.json

```diff
@@ -0,0 +1,46 @@
+{
+    "categories": [
+        {
+            "title": "## Breaking Change",
+            "labels": ["break"]
+        },
+        {
+            "title": "## Feature",
+            "labels": ["feat"]
+        },
+        {
+            "title": "## Bug Fix",
+            "labels": ["fix"]
+        },
+        {
+            "title": "## Platform",
+            "labels": ["platform"]
+        },
+        {
+            "title": "## Documentation",
+            "labels": ["docs"]
+        }
+    ],
+    "sort": {
+        "order": "ASC",
+        "on_property": "title"
+    },
+    "template": "${{CHANGELOG}}",
+    "pr_template": "- ${{TITLE}} - #${{NUMBER}}",
+    "empty_template": "- no noteworthy changes",
+    "label_extractor": [
+        {
+            "pattern": "([^\\(]+)\\(.+\\): .+",
+            "on_property": "title",
+            "target": "$1"
+        }
+    ],
+    "transformers": [
+        {
+            "pattern": "([^\\(]+)\\(?([^\\)]+)?\\)?: (.+)",
+            "target": "- **$2:** $3"
+        }
+    ],
+    "max_pull_requests": 100,
```

Comment:
Probably not so important but just curious - isn't every release created after each PR is merged?
What's the purpose of this limit? 🤔

Reply:
nono, the GitHub release is only created once per day at night. So, I chose just a number, which is high enough, but not to crazy 😄